### DISCUSSION TITLE
Can not update/refresh preview when emebdding codesandbox

### DISCUSSION THREAD

**Discussion by hilmanski** [codesandbox/codesandbox-client#7202] [Q&A]
Hi 
currently using codesandbox by embedding it on my site.
The preview is not refreshing (even with button) .
Nothing happen when I update the code.

The only error I saw is

![Screen Shot 2022-12-01 at 14 45 34](https://user-images.githubusercontent.com/4522467/204984680-241bea86-cded-439f-a45d-9b96fb3dd274.png)

**Comment by tristandubbeld** [comment]
Hey @hilmanski, thanks for letting us know. Do you have more details to share with us, like the sandbox url or your site? This makes it easier for us to debug what the issue might be. Thanks!

**Reply by hilmanski** [reply]
Sure sorry for not giving the detail.
Here's my link: https://koding-pi.vercel.app/html/menulis/
*It's also happen on localhost.

My iframe


 
--
<iframe src="https://codesandbox.io/embed/koding-html-intro-1s28xs?autoresize=1&fontsize=14&hidenavigation=1&theme=dark&hidedevtools=1&forcerefresh=1&view=split"
  | style="width:100%; height:300px; border:0; border-radius: 10px; overflow:hidden;"
  | title="koding-html-intro"
  | sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
  | ></iframe>
  |

**Reply by tristandubbeld** [reply]
Thanks! We're looking into it. It has something to do with your sandbox being static (no bundler). If there are any further developments and if it is possible to make it editable, we'll let you know. In the meantime, depending on what you want to accomplish, you could create a non-static sandbox or [take a look at Sandpack](https://sandpack.codesandbox.io/) to see if it fits your needs.

**Reply by hilmanski** [reply]
Hi @tristandubbeld 
1. do you mean currently embed sandbox not working for 'vanilla' HTML ?
2. Hi I'd like to use Sandpack, but it looks like used for React. There's no clear documentation on how to use it on vanilla javascript. On my case , I'm using it for a Hugo (static site generator) (non React environment)

**Reply by tristandubbeld** [reply]
> do you mean currently embed sandbox not working for 'vanilla' HTML?

Yes, it seems like that is the case. When embedding, the server bundles the sandbox and serves the static files for the embed which are, in this case, static too. Sadly this isn't anything we can fix in the short term.

> I'd like to use Sandpack, but it looks like used for React.

That's true. In this case I think there might be another way to circumvent using Sandpack or the embed, by embedding the whole editor. If you replace `/embed/` with `/s/` in the `iframe` `src` you should be good to go.

```
<iframe src="https://codesandbox.io/s/koding-html-intro-1s28xs?autoresize=1&fontsize=14&hidenavigation=1&theme=dark&hidedevtools=1&forcerefresh=1&view=split"   | style="width:100%; height:300px; border:0; border-radius: 10px; overflow:hidden;"   | title="koding-html-intro"   | sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"   | >
```

**Reply by hilmanski** [reply]
Thanks @tristandubbeld appreciate your help and guide
